class-linklist.php

```diff
@@ -139,6 +139,94 @@ public function get_fields() : array {
 								],
 							],
 						],
+						[
+							'key'        => $this->field_key . '_flex_dynamic',
+							'name'       => 'dynamic',
+							'label'      => esc_html__( 'Dynamic selection', 'hogan-linklist' ),
+							'display'    => 'block',
+							'sub_fields' => [
+								[
+									'key'   => $this->field_key . '_dynamic_list_heading',
+									'label' => esc_html__( 'Heading', 'hogan-linklist' ),
+									'name'  => 'list_heading',
+									'type'  => 'text',
+								],
+								[
+									'type'          => 'select',
+									'key'           => $this->field_key . '_flex_dynamic_list',
+									'label'         => __( 'Content Type', 'hogan-linklist' ),
+									'name'          => 'dynamic_list_content_type',
+									'instructions'  => __( 'Select the content type to build linklist from', 'hogan-linklist' ),
+									'required'      => 1,
+									'wrapper'       => [
+										'width' => '50',
+									],
+									'choices'       => apply_filters( 'hogan/module/linklist/dynamic_content_post_types', [
+										'post' => __( 'Posts', 'hogan-linklist' ),
+										'page' => __( 'Pages', 'hogan-linklist' ),
+										'tax'  => __( 'Taxonomy pages', 'hogan-linklist' ),
+									], $this ),
+									'return_format' => 'value',
+								],
+								[
+									'type'          => 'number',
+									'key'           => $this->field_key . '_number_of_items',
+									'label'         => __( 'Number of items', 'hogan-linklist' ),
+									'name'          => 'number_of_items',
+									'instructions'  => __( 'Set the number of items to display', 'hogan-linklist' ),
+									'required'      => 1,
+									'default_value' => 3,
+									'min'           => 1,
+									'max'           => 10,
+									'step'          => 1,
+									'wrapper'       => [
+										'width' => '50',
+									],
+								],
+								[
```

Comment:
Jeg tror ikke begge disse feltene `taxonomy_name` og `posts_term` skal med. Er vel kun `posts_term`?

Reply:
`taxonomy_name` er en dropdown med taxonomier, for å liste ut lenker til taxonomy pages. `posts_term`  er en dropdown med termer, for velge hvilken term man skal liste ut poster for.